extensions/email/src/main/java/org/mule/extension/email/internal/sender/SenderOperations.java

```diff
@@ -60,98 +60,97 @@
     @Summary("Sends an email message")
     public void send(@Connection SenderConnection connection,
                      @UseConfig SMTPConfiguration configuration,
-                     EmailContent content, // TODO: create a transformer from string to EmailContent when the sdk have support for it - MULE-9181.
+                     @Optional(defaultValue = PAYLOAD) EmailContent content, // TODO: create a transformer from string to EmailContent when the sdk have support for it - MULE-9181.
```

Comment:
this doesn't make much sense... No that we have the concept of Email, what's the point in communicating the content and the other attributes separately? why not just passing an entire Email object?